elephant/signal_proc.py

```diff
@@ -0,0 +1,130 @@
+'''
+signal_proc module
```

Comment:
I added a Butterworth filtering function in "signal" module, which was meant as a module of signal processing functions.
We need to agree on the naming of this module and use it consistently.